src/docs/tutorial-branding/first-edit/index.md

```diff
@@ -1,63 +1,167 @@
-// Also --marketplaceColorDark and --marketplaceColorLight are generated from this one
-// by adding +/- 10% to lightness.
-export const marketplaceColor = '#7c3aed';
+In this section, you'll change the background colour of an existing
+component on the landing page. To do that, we will locate the component
+we want to edit through the browser's developer tools and make a change
+in the CSS file of that component.
+
+By default, the background colour of the sections on the landing page
+alternates between white and grey. Our goal is to change the colours to
+alternate between grey and a light pastel purple.
+
+Let's start by powering up the development environment by running:
+`yarn run dev`
+
+After that, you should be able to see your marketplace in your browser
+by navigating to http://localhost:3000
+
+Next, you'll want to inspect the element you want to modify using the
+developer console in your web browser.
+
+![Inspect element through the browser's developer tools](./inspect-element.png)
+
+<info>
+
+Inspecting elements using your browser's developer tools is a helpful
+way of locating the correct component to edit when developing your
+marketplace.
+
+</info>
+
+The element you are looking for is a `<div>` with the class name
+`SectionContainer_root` and id `how-it-works-for-customers`. You'll
+notice that the class name has some additional characters appended to it
+like `SectionContainer_root_A1BjA`. That is the class name generated by
+CSS Modules. The random characters added to the end of the class name
+guarantee no clashes between classes, even if you use the same class
+name to style another component elsewhere in your codebase.
+
+Once you've identified the component you want to edit, you can deduce
+the correct CSS file based on the class name. The first part of the
+class name, `SectionContainer`, tells us which React component we want
+to edit. This information makes it easy for us to locate the correct CSS
+file, as all CSS files follow the same naming scheme:
+`SectionContainer.module.css`
+
+Alternatively, you can also directly identify the CSS file you need to
+edit by using the developer tools:
+![Identify the CSS file using the developer tools](./identifycss.png)
+
+The second part of the class name, `root` tells us which CSS selector we
+want to edit. In `SectionContainer.module.css` we next identify the
+.root selector:
+
+```css
+.root {
+  background-size: cover;
+  background-position: center center;
+  background-repeat: no-repeat;
+  width: 100%;
+  position: relative;
+
+  &:nth-of-type(odd) {
+    background-color: var(--colorWhite);
+  }
+}
+```
+
+The &:nth-of-type(odd) selector targets elements that are the
+odd-numbered siblings within their parent container. Therefore, the
+background-colour property is applied to every second element on the
+landing page.
+
+Let’s change the background colour to a new colour. First, define a new
+variable:
+
+```css
+var(—colorPastelPurple)
```

Comment:
I'm confused by the instructions here 😄
- where do I define the variable
- why is the variable `-colorPastelPurple` but the var used in the class below is `--colorPurple`
@shareoc would you be able to clarify this?

Reply:
I'm currently going through the article and changing the colours to match the new Sauna time theme so it's WIP! @SariSaar